app/helpers/application_helper.rb

```diff
@@ -154,4 +154,52 @@ def boolean_radio_buttons_collection(text_vals = { true: 'Yes', false: 'No' })
     [ [true, t(text_vals[:true])], [false, t(text_vals[:false])] ]
   end
 
```

Comment:
These methods are not used by the majority of the application views and/or controllers, so I don't think they belong in this class.

Since these methods _are_ common to 2 views/controllers (User, Company) then you should refactor them into either a module or a common parent class for them.  (My vote = a mixin via a module, vs. inheritance via a parent class)

(  Having 'just' 2 classes that share commonality, I'm fine with using if/else statements to choose which classes to instantiate/which logic to use.  But with more than 2 classes we'd definitely want to make this into separate classes (factories, etc) instead of getting into if/then or case statements for all of these similar checks.

Even more importantly, though, since this is a first implementation, best to keep things simple and refactor later. :-)  )

Reply:
Did you mean "methods" or "method"?  (`model_errors_helper` is used in many places).

In any case, `boolean_radio_buttons_collection` is currently only used in one view.  However, it is designed to be useful for any set of radio buttons, and there is at least one place right now which could be refactored to use this method.

Also, this method is in the current code base, and is not introduced with this PR (we need to limit ancillary changes for this PR as it is enormous enough).